changelog.md

```diff
@@ -1,7 +1,21 @@
 # Change Log
 
-## Unreleased
-* Add: pydantic_autodoc to sphinx build and reconfigure api structure - p1 (@lwasser, #326)
+## v1.3.0rc0
+
+### Added
```

Comment:
Gosh now that i'm looking at this - is it added or is it that it's a massive CHANGE. i think maybe a bunch of these are changes rather than additions. thoughts?

Reply:
also free free to add anything that i missed.

Reply:
Haha, I realize I forgot to add all of these to the changelog, as these PRs were on the feature branch.

> is it added or is it that it's a massive CHANGE

Hard to tell. Individually, most of these PRs changed a few classes and added some backward compatibility support. As a whole, this is of course a big change. But how you added them to the changelog is fine.

Reply:
ok great!